### PR TITLE
Bug/ind active priority

### PR DESCRIPTION
**Pull request by lheagy**
- ensure `indActive` set prior to cell weights in the regularization init so the validator looks for cell weights that are the same size as the indactive
- fixes the bug @dccowan was encountering in the example he generated in #684

### CODE REVIEW

File: tests/base/test_regularization.py [hunk 1]
```diff
@@ -357,5 +357,26 @@ def test_nC_residual(self):
...
+
+        # z direction
+        npad = 12
+        temp = np.logspace(np.log10(1.), np.log10(12.), 19)
+        temp_pad = temp[-1] * 1.3 ** np.arange(npad)
+        hz = np.r_[temp_pad[::-1], temp[::-1], temp, temp_pad]
+        mesh = Mesh.CylMesh([hx, 1, hz], '00C')
+        active = mesh.vectorCCz < 0.
+
+        active = mesh.vectorCCz < 0.
```
Comment (fourndo) [inline tests/base/test_regularization.py]: Duplicate line

Comment (lheagy) [reply tests/base/test_regularization.py]: thanks!

**Comment by lheagy** [comment]
Thanks @fourndo, I have removed the duplicate and cleaned out some commented code in the tests

**Comment by lheagy** [comment]
@dccowan: it looks like you merged a bunch of stuff from your active branch here. I will start a new pr instead.

**Pull request closed without merging**